### PR TITLE
Update changes for NVMe Drives

### PR DESCRIPTION
**Pull request by dnarain-astro**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -48,10 +48,13 @@ If you have any feedback, please submit it to the [Astro Cloud IDE product porta
 
 ### Write to temporary storage on AWS clusters
 
-AWS clusters that use `m5d` and `m6id` worker types can now run tasks which require writing data to ephemeral storage. These worker types now have NVMe SSD volume mounts that can be utilized by tasks. See [Amazon EC2 M6i Instances](https://aws.amazon.com/ec2/instance-types/m6i/) and [Amazon EC2 M5 Instances](https://aws.amazon.com/ec2/instance-types/m5/) for the amount of available storage in each worker type.
+AWS clusters that use `m5d` and `m6id` worker types can now use the full capacity of the NVMe SSD volumes available in these instances. Prior to this release, Astro used the utilization of the root volume to make scheduling decisions; with this change  the entire storage capacity is examined.
```
Comment (kwcanuck) [inline astro/release-notes.md:51]:
```suggestion
AWS clusters that use `m5d` and `m6id` worker types now use the full capacity of the available NVMe SSD volumes. Previously, Astro used only the root volume when making scheduling decisions. 
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -48,10 +48,13 @@ If you have any feedback, please submit it to the [Astro Cloud IDE product porta
 
 ### Write to temporary storage on AWS clusters
 
-AWS clusters that use `m5d` and `m6id` worker types can now run tasks which require writing data to ephemeral storage. These worker types now have NVMe SSD volume mounts that can be utilized by tasks. See [Amazon EC2 M6i Instances](https://aws.amazon.com/ec2/instance-types/m6i/) and [Amazon EC2 M5 Instances](https://aws.amazon.com/ec2/instance-types/m5/) for the amount of available storage in each worker type.
+AWS clusters that use `m5d` and `m6id` worker types can now use the full capacity of the NVMe SSD volumes available in these instances. Prior to this release, Astro used the utilization of the root volume to make scheduling decisions; with this change  the entire storage capacity is examined. 
 
-You can use this storage for simple operations such as a disk-based merge sort or checkpointing to prevent crashes. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md). 
+Tasks using KubernetesPodOperator can create  mounts to any path, but other tasks must write to `/ephemeral` to use this storage.
```
Comment (kwcanuck) [inline astro/release-notes.md:53]:
```suggestion
Tasks using KubernetesPodOperator can create mounts to any path, but all other tasks must write to `/ephemeral` .  
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -48,10 +48,13 @@ If you have any feedback, please submit it to the [Astro Cloud IDE product porta
...
+AWS clusters that use `m5d` and `m6id` worker types can now use the full capacity of the NVMe SSD volumes available in these instances. Prior to this release, Astro used the utilization of the root volume to make scheduling decisions; with this change  the entire storage capacity is examined. 
 
-You can use this storage for simple operations such as a disk-based merge sort or checkpointing to prevent crashes. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md). 
+Tasks using KubernetesPodOperator can create  mounts to any path, but other tasks must write to `/ephemeral` to use this storage.  
+
+**Note that non-KPO tasks that used local storage on these instances will need to be adjusted to write to `/ephemeral` instead.**
```
Comment (kwcanuck) [inline astro/release-notes.md:55]:
```suggestion
**Non-KPO tasks that use local storage will need to be updated to write to `/ephemeral`.**
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -48,10 +48,13 @@ If you have any feedback, please submit it to the [Astro Cloud IDE product porta
...
 
+You can use this storage for simple operations such as a disk-based merge sort or checkpointing to prevent crashes. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md).
```
Comment (kwcanuck) [inline astro/release-notes.md:57]:
```suggestion
You can use temporary storage for simple operations such as a disk-based merge sort or checkpointing. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md). 
```

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md (hunk 1 above)
Comment (jwitz) [reply astro/release-notes.md:51]: @kwcanuck @dnarain-astro Can we condense this change to be a bug fix? The additional docs for how this storage is configured is already set to publish in https://github.com/astronomer/docs/pull/1404/

**Review by jwitz — CHANGES_REQUESTED** [review]
I think we're giving the wrong impression that this is a feature, when in reality it's a bug fix right? We should:
- Condense the note to something similar that I had in  https://github.com/astronomer/docs/pull/1455
- Rely on https://github.com/astronomer/docs/pull/1404 to explain how to configure the feature. The info about using `/ephemeral` should not live in release notes exclusively.

File: astro/release-notes.md (hunk 1 above)
Comment (kwcanuck) [reply astro/release-notes.md:51]: @jwitz makes sense to me.

**Comment by jwitz** [comment]
Thanks for getting this going @dnarain-astro . Based on your comment in https://github.com/astronomer/docs/pull/1455, I'm going to close this out in favor of https://github.com/astronomer/docs/pull/1404 where I've updated the release note in question.

**Comment by kwcanuck** [comment]
@jwitz I'm closing this PR. It appears the issue has been documented in the Release Notes and a solution is provided elsewhere in the documentation. Feel free to reopen this if you think additional work is required.

**Pull request closed without merging**